src/styles/blocks/star.scss

```diff
@@ -0,0 +1,16 @@
+.stars {
+  display: flex;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    background-image: url('../images/star.svg');
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-right: 4px;
+  }
+}
+
+.stars--rating-4 .stars__star:nth-child(-n + 4) {
+  background-image: url('../images/star-active.svg');
+}
```

Comment:
According to [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), modifier class names should be separated by only one dash. In this case, 'stars--rating-4' should be 'stars-rating-4'.